creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -119,6 +119,38 @@ spec:
       networkName: ovs-vlan-100
 ----
 
+It is also possible to define a multus network as the default pod network with
+https://github.com/intel/multus-cni[Multus]. A version of multus after this
+https://github.com/intel/multus-cni/pull/174[Pull Request] is required
+(currently master).
+
+
+*Note the following:*
+
+* A multus default network and a pod network type are mutually exclusive.
+* The virt-launcher pod that starts the VMI will *not* have the pod network
+configured.
+* The multus delegate chosen as default *must* return at least one IP address.
+
+
+The following defines a VMI with an OVS network as the default.
```

Comment:
This example is little misleading since we did not implement IPAM in the OVS plugin.

You do not define any NetworkAttachmentDefinition in this example, could you please drop the OVS mention?

Reply:
Sure

Reply:
Added an OVS NAD with IPAM for the example.

Reply:
What I meant is, that kubevirt/ovs-cni plugin does not support IPAM, it is only L2. If you give it IPAM section, it either fails or ignores it.

Could you maybe use linux bridge instead https://github.com/containernetworking/plugins/tree/master/plugins/main/bridge ?

Reply:
Sorry about that, I'll get right on it.

Reply:
@phoracek how does it look now? (I've actually tested this one on a real setup to verify functionality).